src/main/java/TaskManager.java

```diff
@@ -0,0 +1,146 @@
+import java.util.Scanner;
+
+public class TaskManager {
+    private static final int MAX_SIZE = 100;
+
+    private static final String COMMAND_BYE = "bye";
+    private static final String COMMAND_LIST = "list";
+    private static final String COMMAND_TODO = "todo";
+    private static final String COMMAND_DEADLINE = "deadline";
+    private static final String COMMAND_EVENT = "event";
+    private static final String COMMAND_DONE = "done";
+
+    private static final String SEPARATOR_SPACE = " ";
+    private static final String SEPARATOR_BY = " /by ";
+    private static final String SEPARATOR_AT = " /at ";
+
+    private static final String DIVIDER = "    ____________________________________________________________";
+
+    private Task[] tasks;
+    private int numberOfTasks;
+
+    public TaskManager() {
+        this.tasks = new Task[MAX_SIZE];
+        this.numberOfTasks = 0;
+    }
+
+    public void processUserInput() {
+        Scanner sc = new Scanner(System.in);
+        while(true) {
+            String userInput = sc.nextLine();
+            if(userInput.equals(COMMAND_BYE)) {
+                break;
+            }
+
+            if(userInput.equals(COMMAND_LIST)) {
+                printAllTasks();
+            } else if (userInput.startsWith(COMMAND_DONE)) {
+                String[] inputs = userInput.split(SEPARATOR_SPACE);
+                int taskDoneNumber = Integer.parseInt(inputs[1]);
+
+                setTaskDone(taskDoneNumber - 1);
+            } else {
+                addTask(userInput);
+            }
+        }
+    }
+
+    private void addTask(String taskInput) {
+        if(taskInput.startsWith(COMMAND_TODO)) {
```

Comment:
```suggestion
        if (taskInput.startsWith(COMMAND_TODO)) {
```
Noticed a coding standard violation error here